packages/deck-scripts/config/baseRollupConfig.js

```diff
@@ -0,0 +1,63 @@
+const { nodeResolve } = require('@rollup/plugin-node-resolve');
```

Comment:
This file is moved over from `packages/pluginsdk/pluginconfig/rollup.config.js`. But since I made some changes to the other file, git doesn't treat this as a move :(.

Reply:
It's worth splitting this up into two commits.  move the file in one commit, and don't make any other changes.  Then make changes in a successive commit.